use-cases/assembler_metal/config/default.json

```diff
@@ -0,0 +1,76 @@
+{
+    "zosmfHost": "your.zosmf.instance.here",
+    "zosmfPort": 443,
+    "zosmfUser": "youruid",
+    "zosmfPassword": "yourpass",
+    "basepath": null,
+    "rejectUnauthorized": false,
+    "job": {
+        "name": "TEMPLATE",
+        "account": "#ACCT"
+    },
+    "program": {
+        "name": "TEMPLATE"
+    },
+    "asmDataset": {
+        "createDataset": true,
+        "dsn": "YOURID.PUBLIC.WORK.ASMPGM",
+        "member": "TEMPLATE",
+        "blocksize": 6160,
+        "directoryblocks": 25,
+        "alcunit": "CYL",
+        "primary": 9,
+        "secondary": 1,
+        "lrecl": 80,
+        "dsorg": "PO",
+        "volser": null
+    },
+    "asmmacDataset": {
+        "createDataset": true,
+        "dsn": "YOURID.PUBLIC.WORK.ASMMAC",
+        "blocksize": 6160,
+        "directoryblocks": 25,
+        "alcunit": "CYL",
+        "primary": 9,
+        "secondary": 1,
+        "lrecl": 80,
+        "dsorg": "PO",
+        "volser": null
+    },
+    "adataDataset": {
+        "createDataset": true,
+        "dsn": "YOURID.PUBLIC.WORK.SYSADATA",
+        "blocksize": 6160,
+        "directoryblocks": 25,
+        "alcunit": "CYL",
+        "primary": 9,
+        "secondary": 1,
+        "lrecl": 80,
+        "dsorg": "PO",
+        "volser": null
+    },
+    "objlibDataset": {
+        "createDataset": true,
+        "dsn": "YOURID.PUBLIC.WORK.OBJLIB",
+        "blocksize": 9140,
```

Comment:
Should "blocksize" here be 9040? I get this error when running `npm run start`:
```
Failed to create dataset IBMUSER.PUBLIC.WORK.OBJLIB: z/OSMF REST API Error:
Rest API failure with HTTP(S) status 500
category: 1
rc:       4
reason:   13
message:  For a FB file, the BLKSIZE must be an integer multiple of the LRECL.
```

Reply:
If I change "blocksize" to 9040, the datasets are created successfully but the job fails for me with an S013 Abend.

Based on [IBM doc](https://www.ibm.com/support/pages/how-debug-s013-abends-sci84688), it seems like there is an incorrect DCB for a DD.

The offending DD mentioned in the JES log for me is the SYSIN DD on the ASSEMBLE step:
` 11.07.39 JOB05791  IEC141I 013-18,IGG0191B,IBMUSERJ,ASSEMBLE,SYSIN,2006,TSOC20,  796`

Reply:
@tjohnsonBCM I am changing the default - can you try using 6160?

Reply:
Both 6160 and 9040 work for me for creating the dataset. The Abend was caused by user error 😢 I was using the wrong program/member name.